packages/blade/src/components/Input/SelectInput/SelectInput.tsx

```diff
@@ -53,6 +53,8 @@ const SelectInput = (props: SelectInputProps): JSX.Element => {
         {...baseInputProps}
         as="button"
         textAlign="left"
+        value={value}
+        defaultValue="Select Option"
```

Comment:
Let's do this for now, but once you start working on the Dropdown, make sure to double check this. 

Since in the native HTML `select` you'll need to pass and empty string "" to signify that nothing is selected or selected value is empty.

Reply:
Oh so that hidden input you see here will basically help in doing the form validation (I've kept the value blank there)

The reason being, 
- we don't have access to all options while we're rendering select so it is a bit complex to render a select there. 
- We anyway cover accessibility of combobox on the button and dropdown itself so the select will be aria-hidden
- `input` field is enough for setting the value for us. I've also seen some libraries do this. (react-select does this, radix-ui weirdly doesn't care about form submissions at all lol)

Reply:
Hmm 🤔  okay. 

radix does render native [select & options](https://github.com/radix-ui/primitives/blob/81b25f4b40c54f72aeb106ca0e64e1e09655153e/packages/react/select/src/Select.tsx#L1112) though so it should work as long as we pass the props like isRequired etc. 

react-select on other hand takes an autocomplete based approach where you can also type on the input thus uses input. 

Anyways let's complete the implementation by adding the dropdown, it's hard to do a proper behaviour & accessibility audit without the full implementation ready. After that we can check for bugs & edge cases.

Reply:
> react-select on other hand takes an autocomplete based approach where you can also type on the input thus uses input.

Yup but they also have the select-only ones where also they use input only
<img width="549" alt="image" src="https://user-images.githubusercontent.com/30949385/210743987-21c9a07d-076e-4307-a0dc-cf088ce43b77.png">


> radix does render native [select & options](https://github.com/radix-ui/primitives/blob/81b25f4b40c54f72aeb106ca0e64e1e09655153e/packages/react/select/src/Select.tsx#L1112) though so it should work as long as we pass the props like isRequired etc.

Strange I tried to search for `<select` on their demo page's inspect and couldn't find any 🤔 https://www.radix-ui.com/docs/primitives/components/select

Reply:
> Anyways let's complete the implementation by adding the dropdown, it's hard to do a proper behaviour & accessibility audit without the full implementation ready. After that we can check for bugs & edge cases.

Yup makes sense